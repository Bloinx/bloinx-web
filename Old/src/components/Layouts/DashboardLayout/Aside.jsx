import React from "react";
import PropTypes from "prop-types";
import styles from "./Aside.module.scss";

export default function Aside({ children }) {
  return <div className={styles.Aside}>{children}</div>;
}

Aside.propTypes = {
  children: PropTypes.node.isRequired,
};
