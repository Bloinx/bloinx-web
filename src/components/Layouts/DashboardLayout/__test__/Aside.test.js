import React from 'react';
import renderer from 'react-test-renderer';

import Aside from '../Aside';

describe('Aside', () => {
  it('renders correctly', () => {
    const Foo = () => <div />;
    const tree = renderer.create(
      <Aside><Foo /></Aside>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
