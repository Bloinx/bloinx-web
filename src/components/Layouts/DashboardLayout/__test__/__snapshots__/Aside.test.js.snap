// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Aside renders correctly 1`] = `
<div
  className="Aside"
>
  <div />
</div>
`;
